diesel_derives/src/table.rs

```diff
@@ -690,6 +690,23 @@ fn expand_column_def(column_def: &ColumnDef) -> TokenStream {
     let sql_name = &column_def.sql_name;
     let sql_type = &column_def.tpe;
 
+    let full_join_impl = if cfg!(any(feature = "postgres", feature = "sqlite")) {
```

Comment:
This is separate from `backend_specific_column_impl` as it's both Postgres & SQLite. Not sure if there's different preferred approach though.